verifier/verifier.go

```diff
@@ -198,24 +216,26 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		for _, pc := range pluginCapabilities {
 			// skip the revocation capability if the trust policy is configured to skip it
 			if outcome.VerificationLevel.Enforcement[trustpolicy.TypeRevocation] == trustpolicy.ActionSkip && pc == proto.CapabilityRevocationCheckVerifier {
+				logger.Debugf("Skipping the %v validation", pc)
 				continue
 			}
 			capabilitiesToVerify = append(capabilitiesToVerify, pc)
 		}
 
 		if len(capabilitiesToVerify) > 0 {
+			logger.Debugf("Executing verification plugin with capabilities %v", capabilitiesToVerify)
```

Comment:
Although we mentioned "Finding" the plugin with pluginName above, we haven't printed out the result of it. We can add the plugin name here just like in Sign.

Reply:
Updated.